src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/VepVersionValidator.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2015-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.parameters.validation;
+
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersInvalidException;
+import org.springframework.batch.core.JobParametersValidator;
+
+import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
+
+/**
+ * Checks that the VEP version option has benn filled in
+ *
+ * @throws JobParametersInvalidException If the vep version is null or empty
+ */
+public class VepVersionValidator implements JobParametersValidator {
+    @Override
+    public void validate(JobParameters parameters) throws JobParametersInvalidException {
+        ParametersValidatorUtil
+                .checkIsValidString(parameters.getString(JobParametersNames.APP_VEP_VERSION),
```

Comment:
What are we getting if we use integers? I would prefer to stick with strings, as it's understandable that they used something like 88.1 or to note the migration from ensembl-tools to ensembl-vep